### PR TITLE
[Win32] Add setAccentColor

### PR DESCRIPTION
**Pull request by floppyhammer**
Also fixed a color issue in `getAccentColor`.

### CODE REVIEW

File: src/win32_window.c [hunk 1]
```diff
@@ -65,19 +74,43 @@ static int getAccentColor(float color[4])
...
+    );
+
+    DwmSetWindowAttribute(
+            handle, DWMWA_CAPTION_COLOR,
+            &accentColor, sizeof(accentColor)
+    );
+
+    return GLFW_TRUE;
+}
+
```
Comment (ws909) [inline src/win32_window.c:116]: How do you reset this color to the defaults?

According to the documentation: 

> The app is responsible for changing the border color according to state changes, such as a change in window activation. 

How can GLFW handle that?

Comment (floppyhammer) [reply src/win32_window.c:116]: @ws909 Good question. I wasn't aware of that. I have added a new function to do that. A dedicated function might not be necessary, as we can achieve the same result by passing a specific color to `setAccentColor`. It's up to you.

Comment (ws909) [reply src/win32_window.c:116]: Thanks! :)

One more question: would it suffice to observe `WM_SETTINGCHANGE` in the helper window’s event listening function to support the system theme changed callback?

**Review by ws909 — APPROVED** [review]

**Comment by floppyhammer** [comment]
@ws909 There's also `WM_THEMECHANGED`, but I don't find any scenarios in which it gets hit. So, I suppose `WM_SETTINGCHANGE` will suffice.

**Comment by ws909** [comment]
Thanks. :D Will have a closer look in a day or two. Everything seems good to me, but I can’t compile and test it on Windows at the moment. And when I do, I’ll be on Windows 10, and this seems to only be available on Windows 11. 
Thanks for all your help. :) Might redesign the API quite a bit, though, but the implementation will stay mostly similar. 
Camilla has given her support to some kind of a theming API, but I don’t think this’ll make it into 3.4.

**Review by ws909 — APPROVED** [review]

**Pull request merged**